docker/jupyter_notebook_config.py

```diff
@@ -0,0 +1,17 @@
+# pulled from https://hub.docker.com/r/ansingh7115/leonardo-notebooks/
+# and added base_url configs for local proxy testing
+
+from jupyter_core.paths import jupyter_data_dir
+import subprocess
+import os
+import errno
+import stat
+
+c = get_config()
+c.NotebookApp.ip = '*'
+c.NotebookApp.port = 8000
+c.NotebookApp.open_browser = False
+c.NotebookApp.base_url = '/notebooks/test/'
```

Comment:
Ah, I see. That being the case I'm okay with this. It doesn't really affect the user's experience, since all they see is the Leo URL anyway.